package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@playkit-js/playkit-js-kaltura-cuepoints",
+  "version": "1.0.0",
+  "main": "dist/playkit-kaltura-cuepoints.js",
+  "scripts": {
+    "clean": "rm -rf ./dist",
+    "prebuild": "npm run clean",
+    "build": "webpack --mode production",
+    "dev": "webpack-dev-server --mode development",
+    "watch": "webpack --progress --colors --watch --mode development",
+    "test": "NODE_ENV=test karma start --color --mode development",
+    "release": "standard-version",
+    "pushTaggedRelease": "git push --follow-tags --no-verify origin master",
+    "eslint": "eslint . --color",
+    "commit:dist": "git add --force --all dist && (git commit -m 'chore: update dist' || exit 0)",
+    "docs:generate": "documentation build flow-typed/** src/** -f md -o docs/configuration.md",
+    "docs:serve": "documentation serve flow-typed/** src/** --watch",
+    "precommit": "lint-staged",
+    "prettier:fix": "prettier --write ."
+  },
+  "devDependencies": {
+    "@types/node": "^15.0.1",
+    "@typescript-eslint/eslint-plugin": "2.34.0",
+    "@typescript-eslint/parser": "2.34.0",
+    "chai": "^4.2.0",
+    "conventional-github-releaser": "3.1.3",
+    "cross-env": "^7.0.2",
+    "css-loader": "^4.0.0",
+    "documentation": "latest",
+    "husky": "^4.2.5",
+    "istanbul": "^0.4.5",
+    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#FEV-985-ts-definitions",
```

Comment:
I think we can use `master` here

Reply:
Sure, that was before I merged it to Master. Will change it now, thanks